docs/standard/security/generating-keys-for-encryption-and-decryption.md

```diff
@@ -10,73 +10,74 @@ helpviewer_keywords:
+Dim aes As Aes = Aes.Create()  
 ```  
   
 ```csharp  
-TripleDESCryptoServiceProvider tdes = new TripleDESCryptoServiceProvider();  
+Aes aes = Aes.Create();  
 ```  
   
  When the previous code is executed, a new key and IV are generated and placed in the **Key** and **IV** properties, respectively.  
   
  Sometimes you might need to generate multiple keys. In this situation, you can create a new instance of a class that implements a symmetric algorithm and then create a new key and IV by calling the **GenerateKey** and **GenerateIV** methods. The following code example illustrates how to create new keys and IVs after a new instance of the symmetric cryptographic class has been made.  
   
 ```vb  
-Dim tdes As TripleDESCryptoServiceProvider = new TripleDESCryptoServiceProvider()  
-tdes.GenerateIV()  
-tdes.GenerateKey()  
+Dim aes As Aes = Aes.Create()  
+aes.GenerateIV()  
+aes.GenerateKey()  
 ```  
   
 ```csharp  
-TripleDESCryptoServiceProvider tdes = new TripleDESCryptoServiceProvider();  
-tdes.GenerateIV();  
-tdes.GenerateKey();  
+Aes aes = Aes.Create();  
+aes.GenerateIV();  
+aes.GenerateKey();  
 ```  
   
- When the previous code is executed, a key and IV are generated when the new instance of **TripleDESCryptoServiceProvider** is made. Another key and IV are created when the **GenerateKey** and **GenerateIV** methods are called.  
+ When the preceding code is executed, a key and IV are generated when the new instance of **Aes** is made. Another key and IV are created when the **GenerateKey** and **GenerateIV** methods are called.
   
-## Asymmetric Keys  
- The .NET Framework provides the <xref:System.Security.Cryptography.RSACryptoServiceProvider> and <xref:System.Security.Cryptography.DSACryptoServiceProvider> classes for asymmetric encryption. These classes create a public/private key pair when you use the parameterless constructor to create a new instance. Asymmetric keys can be either stored for use in multiple sessions or generated for one session only. While the public key can be made generally available, the private key should be closely guarded.  
+## Asymmetric Keys
+
+ .NET provides the <xref:System.Security.Cryptography.RSA> and <xref:System.Security.Cryptography.DSA> classes for asymmetric encryption. These classes create a public/private key pair when you use the parameterless `Create()` method to create a new instance. Asymmetric keys can be either stored for use in multiple sessions or generated for one session only. While the public key can be made generally available, the private key should be closely guarded.  
   
  A public/private key pair is generated whenever a new instance of an asymmetric algorithm class is created. After a new instance of the class is created, the key information can be extracted using one of two methods:  
   
 - The <xref:System.Security.Cryptography.RSA.ToXmlString%2A> method, which returns an XML representation of the key information.  
```

Comment:
I'm not a big fan of this method, so if we stop talking about it here that would make me happy.

There's now also ExportSubjectPublicKeyInfo, ExportPkcs8PrivateKey, and ExportEncryptedPkcs8PrivateKey on all Asymmetricalgorithm classes, which export the key in standard binary formats (X.509 SubjectPublicKeyInfo, PKCS#8 PrivateKeyInfo and PKCS#8 EncryptedPrivateKeyInfo).  Additionally, the RSA class provides algorithm specific ExportRSAPublicKey and ExportRSAPrivateKey (the PKCS#1 RSAPublicKey and PKCS#1 RSAPrivateKey binary formats).

That's a lot of data, and I don't have a strong opinion on how much gets used here.  Presumably whatever makes sense for accompanied samples.